bfm/README.md

```diff
@@ -0,0 +1,25 @@
+# Business File Manager (BFM)
+
+**Anmeldedaten für gemeinsames BFM-Konto**
+
+1. Zu dieser Seite gehen : [BFM](https://my.business-filemanager.de/Login.zul)
+2. Anmeldedaten : 
+
+- E-mail-Adresse : tractusxdays@gmail.com
```

Comment:
Please provide a doubleSlash Email. 
Please get in contact with Kerstin to discuss a better approach to share these files with bfm